hack/local-up-cluster.sh

```diff
@@ -609,7 +611,9 @@ EOF
 fi
 }
 
-test_docker
+if [[ "${CONTAINER_RUNTIME}" == "docker" ]] || [[ -z "${CONTAINER_RUNTIME}" ]]; then
+  test_docker
```

Comment:
@ixdy Yes, but how to test `remote` container-runtime or we just skip test when we use `remote`?